CONTRIBUTING.md

```diff
@@ -273,8 +273,7 @@ Other methods used by some trace modules:
   selections.
 - `_module.convert`: Sometimes separated from `_module.plot` or `_module.calc` to convert the
   plotly.js settings to another framework e.g. to `gl-plot3d` for `gl3d` traces, to
-  `mapbox-gl` from `mapbox` traces. This split can also make the logic easier to
-  test.
+  `mapbox-gl` from `mapbox` traces. This split can make the logic easier to test.
```

Comment:
What pathway calls `convert`? ie is it called by `plot`, by `calc`, or from a higher level of the framework?

Reply:
during `plot`:

- all `gl3d` traces
- all `mapbox` traces
- `heatmapgl`, `contourgl`, `pointcloud`

during `calc`:
- `scattergl`, `scatterpolargl`

Reply:
OK good - the key for contributors is to know it's not hooking into a predefined pathway in the framework.
```suggestion
  `mapbox-gl` from `mapbox` traces. This split can make the logic easier to test.
  If you make a `convert`, you should call it from either `calc` or `plot`.
```